lib/scope.js

```diff
@@ -2330,9 +2339,14 @@ module.exports = {
     }  // ends act depending values
   },  // Ends scope.getThereIsAnotherValue()
 
+  is_on_interview_still: async function ( scope ) {
+    let server_url = session_vars.get_da_server_url();
+    return await scope.page.url().startsWith(server_url);
```

Comment:
I think this might have to get separated into 2 lines for the await to work properly

```suggestion
    let new_url = await scope.page.url();
    return new_url.startsWith(server_url);
```